pkg/contexts/oci/ociutils/helm/ignore/doc.go

```diff
@@ -28,7 +28,7 @@ The formatting rules are as follows:
 
 Example:
 
-	# Match any file named foo.txt
+	# Match exact file name
```

Comment:
This is generated text by helm.

It means, that any occurrence of a file named `foo.txt` is matched, in contrast to /foo.txt, which would match the exact file. I think, this is what this comment want to describe.

Reply:
I would argue with that, if we go down the meaning of the words, `/foo.txt` is an exact path, while `foo.txt` is an exact filename.

But, I'll revert that change.